README.md

```diff
@@ -6,14 +6,15 @@
 
 ## Windows
```

Comment:
This is not quite windows specific :), I think it's actually the Android setup, you can have a section for Windows tips&tricks at the end of the section